docs/model_library/index.rst

```diff
@@ -67,13 +69,25 @@ Optimization Model Notation
           - If 1, :math:`\textcolor{blue}i` wells selected for plugging in project :math:`\textcolor{blue}k`
 
 
+.. list-table:: **Continuous Variables**
+        :widths: 25 75
+        :header-rows: 1
+
+        * - Symbol
+          - Description
+        * - :math:`\textcolor{red}{SLB}`
+          - The slack variable representing the difference between the total budget and the actual budget used
```

Comment:
I think you are using S_{b} in the Parameters table

Reply:
I would prefer not using a subscript as it raises the question of what is the set 'b' is indexing over. Whatever you utilize, please stay consistent throughout the doc

Reply:
I like SLB or some other non-indexed notation because, as Yash pointed out, an index immediately indicates an associated set.